doc/man1/openssl-smime.pod.in

```diff
@@ -397,9 +397,9 @@ Verify a message and extract the signer's certificate if successful:
 
 Send encrypted mail using triple DES:
 
- openssl smime -encrypt -in in.txt -from steve@openssl.org \
+ openssl smime -encrypt -in in.txt -out mail.msg -from steve@openssl.org \
```

Comment:
Can we fix this in code?

Reply:
No, this cannot be easily fixed in code.